recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: uwsgi
```

Comment:
Good point. I just want to iron it out before we add it. Though I like your argument.

@ocefpaf, any thoughts on naming here? Does lowercase seem ok to you?